tests/cnf/ran/oran/internal/helper/clusterinstance.go

```
@@ -0,0 +1,197 @@
+		})
+	}
+
+	nodeGroups := make([]map[string]any, 0, 2)
+	if len(grouped["master"]) > 0 {
+		nodeGroups = append(nodeGroups, map[string]any{
+			"name":  "master",
+			"role":  "master",
+			"nodes": grouped["master"],
+		})
+	}
+	if len(grouped["worker"]) > 0 {
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above if)

(wsl_v5)